kahuna/public/js/search/query.js

```diff
@@ -82,6 +82,9 @@ query.controller('SearchQueryCtrl',
     }
 
     $scope.$watchCollection(() => ctrl.filter, onValChange(filter => {
+        if (filter.query.indexOf('~') === 0) {
```

Comment:
> Or maybe we should only detect it as a collection if the query starts with ~?

I think this only checks if the first character of the query is a '~'. 

If a quote/word begins with '~'  it could still be used as a non-collection query if it was wrapped in quote marks e.g. "~said Alice".

Still, as is, it wont work for a search term "queen ~g2/sweden"